contracts/svm/programs/express_relay/src/lib.rs

```diff
@@ -309,3 +463,94 @@ pub struct WithdrawFees<'info> {
     #[account(mut, seeds = [SEED_METADATA], bump, has_one = admin)]
     pub express_relay_metadata: Account<'info, ExpressRelayMetadata>,
 }
+
+#[derive(AnchorSerialize, AnchorDeserialize, Eq, PartialEq, Clone, Copy, Debug)]
+pub enum FeeToken {
+    Input,
+    Output,
+}
+
+/// For all swap instructions and contexts, input and output are defined with respect to the searcher
+/// So mint_input refers to the token that the searcher provides to the trader
+/// mint_output refers to the token that the searcher receives from the trader
+/// This choice is made to minimize confusion for the searchers, who are more likely to parse the program
+#[derive(AnchorSerialize, AnchorDeserialize, Eq, PartialEq, Clone, Copy, Debug)]
+pub struct SwapArgs {
+    pub amount_input:  u64,
+    pub amount_output: u64,
+    // The referral fee is specified in parts per million (e.g. 100 = 1 bp)
```

Comment:
nit, i'd prefer to keep the label in the variable name as well, just to avoid any confusion and keep the expectation very explicit

Reply:
done, also updated it to bps to be consistent with the other fee ratios that we have